engine/src/main/java/org/terasology/rendering/opengl/BaseFBOsManager.java

```diff
@@ -32,6 +32,8 @@
 
     FBO get(ResourceUrn fboName);
```

Comment:
I don't think so. If you look at the source of this it's quite clear what it gets from the return type. 

If you look at the code using it, it will be (or should be) in the form `get(myFBO)` which is less redundant than `getFBO(myFBO)`.